gtests/net/mptcp/mp_join/mp_join_server_v4.pkt

```diff
@@ -17,22 +17,25 @@
 +0    accept(3, ..., ...) = 4
 
 // add_addr
-+0.0    >                                .  1:1(0)  ack 1             <nop, nop,         TS val 4074418293 ecr 4074418292, add_address addr[saddr1] hmac=auto, dss dack4=1 dll=0 nocs>
-// TODO: send echo bit
-// +0.0 <                                .  1:1(0)  ack 1             <nop, nop,         TS val 4074418293 ecr 4074418292, add_address addr[saddr1],           dss dack8=1 dll=0 nocs>
++0.0    >                                .  1:1(0)  ack 1             <nop, nop,         TS val 4074418293 ecr 4074418292, add_address address_id=1 addr[saddr1] hmac=auto, dss dack4=1 dll=0 nocs>
++0.0    <                                .  1:1(0)  ack 1  win 256                                                        <add_address address_id=1 addr[saddr1] addr_echo, dss dack4=1 dll=0 nocs>
 
-
-+0.2    <                               P.  1:3(2)  ack 1  win 256    <nop, nop,         TS val 4074418292 ecr 4074410674,                mpcapable v1 flags[flag_h] key[skey, ckey] mpcdatalen 2, nop, nop>
++0.0    <                               P.  1:3(2)  ack 1  win 256    <nop, nop,         TS val 4074418292 ecr 4074410674,                mpcapable v1 flags[flag_h] key[skey, ckey] mpcdatalen 2, nop, nop>
```

Comment:
out of curiosity, why changing the timing here? (I think that indeed, we can reduce it, we already sent the echo bit before)

Reply:
just to shorten the test by 0.6 seconds:-)

Reply:
err, just 0.4 since that `_v4` in the script name